webpack/components/extensions/HostDetails/Tabs/TracesTab.js

```diff
@@ -43,20 +84,28 @@ const TracesTab = () => {
         fetchItems={fetchItems}
         autocompleteEndpoint={`/hosts/${hostId}/traces/auto_complete_search`}
         foremanApiAutoComplete
+        displaySelectAllCheckbox
+        selectPage={selectPage}
+        selectNone={selectNone}
+        selectAll={selectAll}
         rowsCount={results?.length}
+        areAllRowsOnPageSelected={areAllRowsOnPageSelected}
+        areAllRowsSelected={areAllRowsSelected}
         variant={TableVariant.compact}
         status={status}
         metadata={meta}
+        selectedCount={selectedTraces.size}
       >
```

Comment:
These new props when defined in other pages should start supporting the multi-page select on those pages correct? Can we also have an option to hide select all across pages dropdown item..Not all APIs will immediately support that and the disabled option might be disabled for some time on those pages..

Reply:
@sjha4 Yeah, this is covered by the`displaySelectAllCheckbox` prop which is false by default :)